custom_components/electrasmart/climate.py

```diff
@@ -114,6 +121,7 @@ def __init__(self, ac, imei, token, use_shared_sid):
         """Initialize the thermostat."""
         self._name = ac[CONF_AC_NAME]
         self.ac = AC(imei, token, ac[CONF_AC_ID], None, use_shared_sid)
+        self._attr_preset_mode = PRESET_NONE
```

Comment:
Regarding this member: I see you're keeping track of the preset mode in `update()`. However you don't seem to be using it anywhere; and I think it makes sense, as the whole design here is that the state is received from the electra service API and stored in the `ac` instance (in operoper).

I would suggest removing this member and the relevant code from `update()`. If I'm missing something here let me know.

Reply:
Removed